app/views/helpers/_user_post.html.erb

```diff
@@ -0,0 +1,18 @@
+<% user.posts.each do |post| %>
```

Comment:
- [ ] The posts that should be rendered on the user show page should be the `recent_posts` in your User model. Kindly rectify that.